spk/davfs2/src/installer.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/sh
+
+# Package
+PACKAGE="davfs2"
+DNAME="davfs2"
+
+DAVUSER="davfs2"
+DAVGROUP="davfs2"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+
+preinst ()
+{
+    exit 0
+}
+
+postinst ()
+{
+    # Link
+    ln -s ${SYNOPKG_PKGDEST} ${INSTALL_DIR}
+
+    # Install busybox stuff
+    ${INSTALL_DIR}/bin/busybox --install ${INSTALL_DIR}/bin
+
+    # Add symlink
+    mkdir -p /usr/local/bin
+    ln -s ${INSTALL_DIR}/sbin/mount.davfs /usr/local/bin/mount.davfs
+    ln -s ${INSTALL_DIR}/sbin/umount.davfs /usr/local/bin/umount.davfs
+    
+    ${INSTALL_DIR}/bin/addgroup ${DAVGROUP}
```

Comment:
Instead of this, please see if you can use the same functions as used for e.g. syncthing. You'll need to adjust other code  accordingly, e.g. add the user to the `users